examples/notepad/build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+import org.jetbrains.compose.compose
+import org.jetbrains.compose.desktop.application.dsl.TargetFormat
+
+plugins {
+    // __KOTLIN_COMPOSE_VERSION__
+    kotlin("jvm") version "1.5.10"
+    // __LATEST_COMPOSE_RELEASE_VERSION__
+    id("org.jetbrains.compose") version (System.getenv("COMPOSE_TEMPLATE_COMPOSE_VERSION") ?: "0.4.0-rc2")
+}
+
+repositories {
+    mavenCentral()
+    maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
+}
+
+dependencies {
+    implementation(compose.desktop.currentOs)
+    implementation(compose.materialIconsExtended)
+}
+
+compose.desktop {
+    application {
+        mainClass = "MainKt"
+        jvmArgs("-Dskiko.rendering.laf.global=true")
```

Comment:
So we no longer set it by default?

Reply:
We don't set in by default in Compose, but can set it by default in our plugin (we never did it before).

@AlexeyTsvetkov, what you think?